install/yamls/ccruntime-peer-pods-in-pod.yaml

```diff
@@ -30,12 +30,20 @@ spec:
             name: peer-pods-cm
         command: ["/usr/local/bin/entrypoint.sh"]
         volumeMounts:
+        - name: ssh
+          mountPath: "/root/.ssh/"
+          readOnly: true
```

Comment:
@snir911 is it possible to have the ssh hostpath volume mount as a kustomize overlay only for libvirt ?

Reply:
right, done